docs/section10/10-3.md

```diff
@@ -0,0 +1,47 @@
+
+**Applicability Checks:**  
+
+1. The hvac system in B-RMR serves a computer room.  
+
+**Function Call:**  
+
+1. get_zones_computer_rooms()  
+2. get_list_hvac_systems_associated_with_zone()  
+
+**Rule Logic:**  
+**Applicability Check 1:**  
+- Call function to get dictonary of zones that include computer rooms and associated square footage in B_RMR: `zones_that_include_comp_rooms_dict_b = = get_zones_computer_rooms(B_RMR)`
+- Get list of the zones that include computer rooms: `zones_that_include_comp_rooms_list_b = zones_that_include_comp_rooms_dict_b.keys()`
+- For each zone_b in zones_that_include_comp_rooms_list_b: `For zone_b in zones_that_include_comp_rooms_list_b:`
+    - Reset does_zone_have_dehumification boolean variable: `does_zone_have_dehumification = FALSE`
+    - Reset has_terminal_heating boolean variable: `has_terminal_heating = FALSE`
+    - Get the hvac system associated with the zone (should only be one, this RDS assumes one HVAC system): `hvac_b = get_list_hvac_systems_associated_with_zone(B_RMR, zone_b.id)`
+    - Check if the zone's maximum_humidity_setpoint_schedule equals Null: `if zone_b.maximum_humidity_setpoint_schedule == Null: does_zone_have_dehumification = FALSE`
+    - Else (i.e. if the zone's maximum_humidity_setpoint_schedule does NOT equal Null): `Else:`
+        - Loop though all elements in the schedule and check if the value equals 1 (convert to an 8760 schedule with a function if needed): `For x in Range(len(zone_b.maximum_humidity_setpoint_schedule.hourly_values))`
+            - Check if the value does not equal one, if it does not then set boolean variable to TRUE: `if zone_b.maximum_humidity_setpoint_schedule.hourly_values[x] != 1: does_zone_have_dehumification = TRUE`
+    - Loop through the terminal units associated with the zone and check if any have heating: `For terminal_b in zone_b.terminals:`
+        - Check if the heating source is Null: `if terminal_b.heating_source == Null: has_terminal_heating = FALSE`
+        - Else if, check if the heating source equals None: `elif terminal_b.heating_source == "None": has_terminal_heating = FALSE`
+        - Else if, check if heating source does not equal none but heating capacity is 0: `elif terminal_b.heating_source != "None" and terminal_b.heating_capacity == 0 : has_terminal_heating = FALSE`
+        - Else, it has terminal heating: `Else: has_terminal_heating = TRUE`
+
+    **Rule Assertion:**
+    - Case 1: For each applicable hvac system (hvac_b above) if the zone served does not have dehumidification then pass: `if does_zone_have_dehumification ==  FALSE: outcome == "PASS"`
+    - Case 2: For each applicable hvac system (hvac_b above) if none of the terminal units have heating then pass:`elif has_terminal_heating == FALSE: outcome == "PASS"`  
+    - Case 3: Else (i.e. HVAC system has dehumification and the system serves computer room space): `Else: outcome == "FAIL"`
```

Comment:
Case 3 text should be (HVAC system that serves computer room space has dehumidification AND reheat

Reply:
Modified